backend/docker_run.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+chown -R $UID:$GID /app
+exec su-exec $UID:$GID /sbin/tini -- node app.js
```

Comment:
This entrypoint should forward args if the cmd starts with - and call a binary otherwise like the official DockerHub Images do.

You can find an example of that [here](https://github.com/SuperSandro2000/docker-images/blob/master/reddiscord/files/entrypoint.sh).